studio/src/components/PluginList/index.tsx

```diff
@@ -5,13 +5,12 @@ const PluginCard = ({ plugin, pluginCallback }:any) => {
     const [expanded, setExpanded] = React.useState(true);
     const { palette } = getTheme();
 
-    const copyToClipboard = (e: Event, text:string) => {
+    const copyToClipboard = (e: React.MouseEvent, text: string, plugin: any) => {
```

Comment:
We should drop the copyToClipboard. That was a hack. Now that we are adding it to the devFooterChatBox

Reply:
We can't drop this. This is being used to add the url to textbox.